edulaw-data/WiredTiger

```diff
@@ -0,0 +1,2 @@
+WiredTiger
```

Comment:
All of these folders should be removed (toss edulaw-data in the gitignore if you haven't already done so)